pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -110,6 +110,9 @@ type VitessClusterSpec struct {
 
 	// TopologyReconciliation can be used to enable or disable registration or pruning of various vitess components to and from topo records.
 	TopologyReconciliation *TopoReconcileConfig `json:"topologyReconciliation,omitempty"`
+
+	// InitReplication can be used to enable or disable the operator initializing replication on a new or restored Vitess Shard object.
+	InitReplication *bool `json:"initReplication,omitempty"`
```

Comment:
I think we should put this in the existing `VitessReplicationSpec` struct since it's another replication-related setting. Since the parent field will already be called `replication`, we won't need to repeat that in the name of the child field. Perhaps we could call it `InitializeMaster` since this is about whether we choose the initial master for a shard that has no master?

Reply:
I agree, although the definition of `VitessReplicationSpec` makes it sound like those options only modify Vitess/MySQL level parameters, whereas our new parameter is an operator action.

Reply:
I can put it in there and slightly modify the description?